kubernetes/services/port_manager.yaml

```diff
@@ -7,23 +7,26 @@ data:
     ignite.thin.client.enable=true
     ignite.host=ignite-alcor-port-service.ignite-alcor-port.svc.cluster.local
     ignite.port=10802
-
     server.max-http-header-size=16384
     server.max-initial-line-length=65536
-
     logging.level.root=DEBUG
-
     microservices.vpc.service.url=http://vpcmanager-service.default.svc.cluster.local:9001
     microservices.subnet.service.url=http://subnetmanager-service.default.svc.cluster.local:9002
     microservices.ip.service.url=http://ipmanager-service.default.svc.cluster.local:9004/ips
     microservices.mac.service.url=http://macmanager-service.default.svc.cluster.local:9005/macs
     microservices.sg.service.url=http://sgmanager-service.default.svc.cluster.local:9008
     microservices.route.service.url=http://routemanager-service.default.svc.cluster.local:9003/routes
+    microservices.router.service.url=http://routemanager-service.default.svc.cluster.local:9003
     microservices.node.service.url=http://nodemanager-service.default.svc.cluster.local:9007/nodes
-    microservices.dataplane.service.url=http://dataplanemanager-service.default.svc.cluster.local:9010/network-configuration
+    microservices.dataplane.service.url=http://dataplanemanager-service.default.svc.cluster.local:9010/v4/port/
```

Comment:
@kevin-zhonghao @cj-chung We may still use V1 APIs in the Medina test cluster. Check this out.

Reply:
Sure